python/MooseDocs/extensions/ifelse.py

```diff
@@ -97,6 +102,19 @@ def hasSubmodule(self, name):
         status = mooseutils.git_submodule_info(MooseDocs.ROOT_DIR, '--recursive')
         return any([repo.endswith(name) for repo in status.keys()])
 
+    def hasLibtorch(self):
+        moose_dir = os.getenv('MOOSE_DIR',
+                              os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', '..')))
+
+        found_libtorch = False
+        # getMooseconfig returns a set of ('ALL', 'True/False'), so we have to iterate through it
+        for it in util.getMooseConfigOption(moose_dir, 'libtorch'):
+            if it.lower() == 'true':
+                found_libtorch = True
+                break
+
+        return found_libtorch
```

Comment:
Just a suggestion so we can create more functions based on config options easily.
```suggestion
    def hasConfigOption(self, option, value):
        moose_dir = os.getenv('MOOSE_DIR',
                              os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', '..')))

        # getMooseconfig returns a set of ('ALL', 'True/False'), so we have to iterate through it
        for it in util.getMooseConfigOption(moose_dir, option):
            if it.lower() == value.lower():
                return True

        return False
```

Reply:
Magnificent idea!